src/applications/validate-mhv-account/containers/Main.jsx

```diff
@@ -7,12 +7,33 @@ import { isLoggedIn, selectProfile } from '../../../platform/user/selectors';
 
 class Main extends React.Component {
   componentDidUpdate(prevProps) {
-    const { loadingProfile, loggedIn } = this.props;
+    const { loadingProfile, loggedIn, mhvAccount, router } = this.props;
+    const prevMhvAccount = prevProps.mhvAccount;
+
     if (prevProps.loadingProfile && !loadingProfile) {
```

Comment:
Should this just be `if (prevProps.loadingProfile && !loadingProfile && !loggedIn)`?